examples/arkode/CXX_serial/ark_advection_diffusion_reaction.hpp

```diff
@@ -830,23 +820,267 @@ int PrintSetup(UserData &udata, UserOptions &uopts)
+        cout << "  advection        = OFF" << endl;
+        cout << "  diffusion        = Explicit" << endl;
+        cout << "  reaction         = Explicit" << endl;
+        break;
+      case(1):
+        // IMEX -- explicit reaction, implicit diffusion
+        cout << "  advection        = OFF" << endl;
+        cout << "  diffusion        = Implicit" << endl;
+        cout << "  reaction         = Explicit" << endl;
+      break;
+      case(2):
+        // IMEX -- explicit diffusion, implicit reaction
+        cout << "  advection        = OFF" << endl;
+        cout << "  diffusion        = Explicit" << endl;
+        cout << "  reaction         = Implicit" << endl;
+        break;
+      case(4):
+        // DIRK -- fully implicit
+        cout << "  advection        = OFF" << endl;
+        cout << "  diffusion        = Implicit" << endl;
+        cout << "  reaction         = Implicit" << endl;
+        break;
+      default:
+        cerr << "ERROR: Invalid splitting option" << endl;
+        return -1;
+        break;
+      }
+    }
+    else
+    {
+      cerr << "ERROR: Invalid problem configuration" << endl;
+      return -1;
+    }
+  }
+  else if (uopts.integrator == 2)
+  {
+    cout << "  integrator       = MRI + ARK" << endl;
+    if (udata.diffusion && udata.advection)
+    {
+      // IMEX slow -- advection-diffusion
+      cout << "  advection        = Slow-Explicit" << endl;
+      cout << "  diffusion        = Slow-Implicit" << endl;
+      if (udata.splitting)
+        cout << "  reaction         = Fast-Implicit" << endl;
+      else
+        cout << "  reaction         = Fast-Explicit" << endl;
+    }
+    else if (!udata.diffusion && udata.advection)
+    {
+      // Explicit slow -- advection
+      cout << "  advection        = Slow-Explicit" << endl;
+      cout << "  diffusion        = OFF" << endl;
+      if (udata.splitting)
+        cout << "  reaction         = Fast-Implicit" << endl;
+      else
+        cout << "  reaction         = Fast-Explicit" << endl;
+    }
+    else if (udata.diffusion && !udata.advection)
+    {
+      cout << "  advection        = OFF" << endl;
+      cout << "  diffusion        = Implicit" << endl;
+      if (udata.splitting)
+        cout << "  reaction         = Fast-Implicit" << endl;
+      else
+        cout << "  reaction         = Fast-Explicit" << endl;
+    }
+    else
+    {
+      // No slow time scale
+      cerr << "ERROR: Invalid problem configuration" << endl;
+      return -1;
+    }
+  }
+  else if (uopts.integrator == 3)
+  {
+    cout << "  integrator       = MRI + CVODE" << endl;
+    // Slow time scale
+    if (udata.diffusion && udata.advection)
+    {
+      // IMEX slow -- advection-diffusion
+      cout << "  advection        = Slow-Explicit" << endl;
+      cout << "  diffusion        = Slow-Implicit" << endl;
+      cout << "  reaction         = Fast-Implicit" << endl;
+    }
+    else if (!udata.diffusion && udata.advection)
```

Comment:
Again, the comments in the header do not mention the possibility of turning off either slow operator (explicit advection or implicit diffusion). Can it be updated accordingly?